patches/0001-Instrument-Mender-client-for-coverage-analysis.patch

```diff
@@ -14,27 +15,50 @@ Also sets the directory for storing the coverage files to '/data/mender/'.
 
 Changelog: None
 
+=======
+From 2304cc8bafca2b7de4f929795124c15e5120978f Mon Sep 17 00:00:00 2001
+From: Ole Petter <ole.orhagen@northern.tech>
+Date: Tue, 27 Apr 2021 16:46:19 +0200
+Subject: [PATCH] Instrument mender binary
+
+Changelog: None
+>>>>>>> 4099759... Fix patch
 Signed-off-by: Ole Petter <ole.orhagen@northern.tech>
 ---
  main.go          | 39 ++++++++++++++++++++++++++++++++++++++-
  system/system.go | 13 +------------
  2 files changed, 39 insertions(+), 13 deletions(-)
 
 diff --git a/main.go b/main.go
+<<<<<<< HEAD
 index 7f46c64..8c4d540 100644
 --- a/main.go
 +++ b/main.go
 @@ -16,12 +16,29 @@ package main
  
  import (
+=======
+index 98e138a..ecb5730 100644
+--- a/main.go
++++ b/main.go
+@@ -18,6 +18,8 @@ import (
+>>>>>>> 4099759... Fix patch
  	"os"
+ 	"os/signal"
+ 	"syscall"
 +	"os/exec"
 +	"time"
  
  	"github.com/mendersoftware/mender/cli"
+<<<<<<< HEAD
  	"github.com/mendersoftware/mender/installer"
  	log "github.com/sirupsen/logrus"
  )
+=======
+@@ -32,6 +34,21 @@ func init() {
+ 	signal.Notify(cli.SignalHandlerChan, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGTERM)
```

Comment:
this function has changed in main.go now it looks like this:
```
signal.Notify(cli.SignalHandlerChan, syscall.SIGUSR1, syscall.SIGUSR2)
```

Reply:
I'll fix this up @NilsOlavKJohansen :)